packages/client/src/__tests__/cloudFunctions.test.ts

```diff
@@ -32,7 +34,46 @@ describe("Cloud functions", () => {
     });
   });
 
-  describe("sendMail", () => {
+  describe("test email sending", () => {
+    // Dummy data for error testing
+    const to = "saul@gmail.com";
+    const subject = "Subject";
+    const html = "html";
+    const smtpServer = createJestSMTPServer({ port: 5000, host: "localhost" });
+
+    afterAll(() => {
+      smtpServer.close();
```

Comment:
If you comment out this line, you'd get more logs including a connection from your machine (this is why I suspect the connection closes too soon) but then you'd have to close the port manually after each run. @silviot